js/rxjs.md

```diff
@@ -0,0 +1,24 @@
+# Bloomfire RxJS Style Guide
+
+*Very much a work in progress*
+
+## Table of Contents
+
+  1. [Observable Naming](#observables)
+
+## observables 
+
+  - Files containing observables should be named in camelCase.
+  - Observable classes should be capitalized and MixedCase.
+  - Observable classes should substitute a trailing $ for the word "observable."
+  - Observable instances should be camelCase.
+
+  ```jsx
+  import SomeAPICall$ from 'app/observables/someAPICallObservable';
+  ````
+
+  ```jsx
+  const someAPICall = app.getInstance(SomeAPICall$);
```

Comment:
I think it's because it's public and you are following it, Chris.

On Wed, May 25, 2016 at 9:34 AM Chris Nixon <notifications@github.com>
wrote:

> In js/rxjs.md
> <https://github.com/bloomfire/styleguides/pull/2#discussion_r64583644>:
>
> > +
> > +  1. [Observable Naming](#observables)
> > +
> > +## observables
> > +
> > +  - Files containing observables should be named in camelCase.
> > +  - Observable classes should be capitalized and MixedCase.
> > +  - Observable classes should substitute a trailing $ for the word "observable."
> > +  - Observable instances should be camelCase.
> > +
> > +  ```jsx
> > +  import SomeAPICall$ from 'app/observables/someAPICallObservable';
> > +  ````
> > +
> > +  ```jsx
> > +  const someAPICall = app.getInstance(SomeAPICall$);
>
> fwiw, I agree that the instance should have a $. Makes the distinction
> between data and streams easier [image: :trollface:] . 👍 for @pretzelface
> <https://github.com/pretzelface> adding standards. Also, for some reason
> I keep getting emails about this repo.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bloomfire/styleguides/pull/2/files/097591b270be8f881f56a78c5d62f7f9a84008df#r64583644>
>
-- 

------------------------
BHEESHMAR REDHEENDRAN <https://www.linkedin.com/in/bheeshmar>
Backend Engineer
*Bloomfire <https://www.bloomfire.com/> *
m 512.658.2547
------------------------
blog <https://bloomfire.com/blog/> | fb <https://www.facebook.com/bloomfire>
 | tw <https://twitter.com/bloomfire> | li
<https://www.linkedin.com/company/bloomfire>